docs/classes/lsp8-identifiable-digital-asset.md

```diff
@@ -104,18 +103,22 @@ await lspFactory.LSP8IdentifiableDigitalAsset.deploy(
     symbol: 'TKN',
     controllerAddress: '0xb74a88C43BCf691bd7A851f6603cb1868f6fc147',
   },
-  { deployReactive: true }
-).subscribe({
-  next: (deploymentEvent) => {
-    console.log(deploymentEvent);
-  },
-  error: (error) => {
-    console.error(error);
-  },
-  complete: () => {
-    console.log('Deployment Complete');
+  {
+    onDeployEvents: {
+      next: (deploymentEvent) => {
+        console.log(deploymentEvent);
+      },
+      error: (error) => {
+        console.error(error);
+      },
+      complete: (digitalAsset) => {
+        console.log('Deployment Complete');
+        console.log(digitalAsset);
```

Comment:
This should also be a contracts object with `LSP7DigitalAsset` as property, to be consistent across all deploy methods

Reply:
and it seems it is this way, so better lets call it `contracts` and log `contracts.LSP8....` to make it obvious